rellic/BC/Util.h

```diff
@@ -37,4 +37,11 @@ bool VerifyModule(llvm::Module *module);
 llvm::Module *LoadModuleFromFile(llvm::LLVMContext *context,
                                  std::string file_name,
                                  bool allow_failure = false);
+
+// Check if an intrinsic ID is an annotation
+bool IsAnnotationIntrinsic(llvm::Intrinsic::ID id);
```

Comment:
`llvm::Intrinsic::ID` is not declared in this header. It is a `typedef unsigned ID;` in the `llvm::Intrinsic` namespace. Forward declare with:

```c++
namespace Intrinsic {
typedef unsigned ID;
}  // namespace Instrinsic
```
up above in the LLVM namespace.